frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -121,6 +122,19 @@ const SearchForm = React.forwardRef<SearchFormRef, SearchFormProps>(
     );
     const methods = useForm<FormValues>({ defaultValues: initialValues });
     const { watch, trigger } = methods;
+    const filterPoolsBySelectedClassification = (
+      allPools: SimplePool[],
+      classification: Maybe<SimpleClassification>,
+    ) =>
+      allPools
+        ?.filter((pool: Maybe<Pick<Pool, "id" | "classifications">>) => {
```

Comment:
Since allPools has a definite type (`SimplePool[]) you don't need the `?` on line 130, and you don't need to define the type of `pool` in the filter callback. Actually, when I changed line 130 to `.filter((pool) => {` typescript found a bug: allPools has `classification` (singular) but inside the filter function we're looking at `classifications` (plural).

Reply:
Also, maybe this filterPoolsBySelectedClassification woud be worth moving to PoolUtils?